### PR TITLE
fix cn doc of alpha_dropout/dropout/dropout2d/dropout3d/npair_loss

### PR DESCRIPTION
**Pull request by huangjun12**
fix cn doc of alpha_dropout/dropout/dropout2d/dropout3d/npair_loss [function/class]

### CODE REVIEW

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image
image
image
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

**Comment by huangjun12** [comment]
image
image

File: doc/paddle/api/paddle/fluid/layers/npair_loss_cn.rst [hunk 1]
```diff
@@ -5,50 +5,33 @@ npair_loss
 
 .. py:function:: paddle.fluid.layers.npair_loss(anchor, positive, labels, l2_reg=0.002)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/npair_loss_cn.rst]: paddle.fluid.layers.npair_loss -> paddle.nn.functional.npair_loss

**Review by TCChenlong — APPROVED** [review]
LGTM

File: doc/paddle/api/paddle/fluid/layers/npair_loss_cn.rst (hunk 1 above)
Comment (huangjun12) [reply doc/paddle/api/paddle/fluid/layers/npair_loss_cn.rst]: Done
